k8s/perftest/common-overlay/bsp/kustomization.yaml

```diff
@@ -4,5 +4,6 @@ namespace: bsp
 bases:
   - ../../../namespaces/bsp
 patchesStrategicMerge:
+  - ../../../namespaces/bsp/blob-router-service/perftest.yaml
```

Comment:
blob router is under reform-scan namespace, it should be there

Reply:
:facepalm: